test/controller.bar.tests.js

```diff
@@ -487,4 +489,154 @@ describe('Bar controller tests', function() {
 		expect(bar._model.borderColor).toBe('rgb(0, 255, 0)');
 		expect(bar._model.borderWidth).toBe(1.5);
 	});
+
+	describe('Bar width', function() {
+		beforeEach(function() {
+			// 2 datasets
+			this.data = {
+				labels: ['January', 'February', 'March', 'April', 'May', 'June', 'July'],
+				datasets: [{
+					data: [10, 20, 30, 40, 50, 60, 70],
+				}, {
+					data: [10, 20, 30, 40, 50, 60, 70],
+				}]
+			};
+		});
+
+		afterEach(function() {
```

Comment:
That's weird, are expectations correctly evaluated since they are outside the `it()` block? I'm not sure tests should occurred in the `beforeEach` or `afterEach` blocks.